package.json

```diff
@@ -98,8 +98,8 @@
     "lodash.get": "4.4.2",
     "lodash.mergewith": "4.6.2",
     "minimist": "1.2.0",
-    "node-sass": "4.12.0",
-    "node-sass-tilde-importer": "1.0.2",
+    "ng-packagr": "5.7.1",
+    "node-sass": "4.13.1",
```

Comment:
While `ng-packagr` doesn't require `node-sass`, we still need to install `node-sass` (instead of relying on "dart" `sass`) because `sass` does not include support for `/deep/` in SCSS files.

Reply:
(We'll support `sass` in Builder 4, however.)